src/custom_hooks/useUrlParams.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
```

Comment:
as we use here React, I believe it is better to have jsx instead of js extension for this file

Reply:
BTW, usually we use snake_case for variables, but files are named with kebab-case